DFC.App.FindACourse/Views/Details/courseDetailsClassroom.cshtml

```diff
@@ -80,69 +58,13 @@
-                    <div class="govuk-accordion__section-summary govuk-body" id="accordion-with-summary-sections-summary-1"> Understand how the course will be delivered. </div>
-                </div>
-                <div id="accordion-default-content-4" class="govuk-accordion__section-content" aria-labelledby="accordion-default-heading-4">
-                    <p id="course-how-you-will-learn">@Html.Raw(Html.HtmlFormat(Model.CourseDetails.HowYoullLearn))</p>
-                </div>
-            </div>}
-        @if (!string.IsNullOrWhiteSpace(Model.CourseDetails.EquipmentRequired)){
-        <div class="govuk-accordion__section">
-            <div class="govuk-accordion__section-header">
-                <h2 class="govuk-accordion__section-heading">  <button type="button" id="need" aria-controls="accordion-default-content-5" class="govuk-accordion__section-button" aria-describedby="accordion-default-summary-5" aria-expanded="false"> What you'll need to bring <span class="govuk-accordion__icon" aria-hidden="true"></span></button></h2>
-                <div class="govuk-accordion__section-summary govuk-body" id="accordion-with-summary-sections-summary-1"> Find out what material you will need. </div>
-            </div>
-            <div id="accordion-default-content-4" class="govuk-accordion__section-content" aria-labelledby="accordion-default-heading-4">
-                <p id="equipment-required">@Html.Raw(Html.HtmlFormat(Model.CourseDetails.EquipmentRequired))</p>
-            </div>
-        </div>
-        }
-    @if (!string.IsNullOrWhiteSpace(Model.CourseDetails.AssessmentMethod))
-    {
-        <div class="govuk-accordion__section">
-            <div class="govuk-accordion__section-header">
-                <h2 class="govuk-accordion__section-heading"> <button type="button" id="assessed" aria-controls="accordion-default-content-6" class="govuk-accordion__section-button" aria-describedby="accordion-default-summary-6" aria-expanded="false"> How you'll be assessed <span class="govuk-accordion__icon" aria-hidden="true"></span></button></h2>
-                <div class="govuk-accordion__section-summary govuk-body" id="accordion-with-summary-sections-summary-1"> Find out more about exams, assignments and assessments. </div>
-            </div>
-            <div id="accordion-default-content-4" class="govuk-accordion__section-content" aria-labelledby="accordion-default-heading-4">
-                <p id="assessment-method">@Html.Raw(Html.HtmlFormat(Model.CourseDetails.AssessmentMethod))</p>
-            </div>
-        </div>
-    }
+        <partial name="AccordionSections/WhoTheCourseIsFor" model="Model" />
```

Comment:
Don't think you need to include `model="Model"` as it is passed by default - could remove it and similar to clean the code a tad